### ISSUE TITLE
Need sound button clicked event

### ISSUE THREAD

**Issue by quedicesebas** [hannadrehman/stories-react#31]
No sure how to achieve it

**Comment by hannadrehman** [comment]
we have a callback that gets called on every story change. u can attach your own sound in that callback.

**Comment by quedicesebas** [comment]
Sorry, I didn't make myself understood. I meant that it would be useful to call an event when the user clicks the sound button in the video stories (for example to keep interaction statistics, see if the button is being seen and so on).

Something like `afterSoundButtonClicked` callback.

**Comment by hannadrehman** [comment]
oh by bad. i didnt quite understand it before. but gotcha. we need to attach a callback for sound button interaction . could be  something like `onSoundChanges(state:boolean)`.  i will add this to our milestones

**Comment by quedicesebas** [comment]
Yes, maybe attach also the storie data or index. If guide me, I can help with that.

**Comment by bugbuddy69** [comment]
same problem, is there no mechanism currently to default the muted functionality to false? If not, I'm willing to contribute in any way

**Comment by hannadrehman** [comment]
we should be able to add new props for story type video. i will add this asap
